neuralmonkey/model/stateful.py

```diff
@@ -0,0 +1,59 @@
+from abc import ABCMeta, abstractproperty
+import tensorflow as tf
+
```

Comment:
Tenhle modul by měl mít docstring, kde se vysvětlí, že tohle jsou abstraktní části modelu, které deklarují, že ty části modelu mají nějaké výstupy a konstruktory jiných částí modelu na to spoléhají.

Reply:
hotovo